staging/src/k8s.io/kubectl/pkg/cmd/delete/delete.go

```diff
@@ -440,3 +519,24 @@ func (o *DeleteOptions) PrintObj(info *resource.Info) {
 	// understandable output by default
 	fmt.Fprintf(o.Out, "%s \"%s\" %s\n", kindString, info.Name, operation)
 }
+
+func (o *DeleteOptions) confirmation(infos []*resource.Info) bool {
+	fmt.Fprintf(o.Out, i18n.T("You are about to delete the following %d resource(s):\n"), len(infos))
+	for _, info := range infos {
+		groupKind := info.Mapping.GroupVersionKind
+		kindString := fmt.Sprintf("%s.%s", strings.ToLower(groupKind.Kind), groupKind.Group)
+		if len(groupKind.Group) == 0 {
+			kindString = strings.ToLower(groupKind.Kind)
+		}
+
+		fmt.Fprintf(o.Out, "%s/%s\n", kindString, info.Name)
+	}
+	fmt.Fprintf(o.Out, i18n.T("Do you want to continue?")+" (y/n): ")
+	var input string
+	_, err := fmt.Fscan(o.In, &input)
```

Comment:
It's probably better to use `fmt.Fscanln` or `r.ReadString('\n')` to ensure that it doesn't hang if we don't specify anything. Currently pressing enter will be stuck until you push any button. 

Can be addressed in followup.